### ISSUE TITLE
Checkbox labels and checkboxes should be associated

### ISSUE THREAD

**Issue by grugnog** [CivicActions/nebula#69]
It's hard/impossible for assistive technology users to know which checkbox is which without this. Here is detail and reference markup:
http://webaim.org/techniques/forms/controls#checkbox

**Comment by grugnog** [comment]
Reopening this - the "label for" part was implemented, but the checkboxes were not given a name parameter, so the labels aren't actually connected with them. See http://webaim.org/techniques/forms/controls#checkbox for example markup.
